docs/fast_data/connectors/debezium_cdc.md

```diff
@@ -526,6 +526,240 @@ debezium.transforms.Reroute.key.enforce.uniqueness=false
 </details>
 
 
+### DB2
+
+In order to receive the executed operations from the DB (following the CDC pattern) you have to [enable Change Data Capture](https://debezium.io/documentation/reference/2.1/connectors/db2.html#setting-up-db2) on your database. To do that, you have to satisfy these requirements:
+- You are logged in to DB2 as the db2instl user.
+- On the DB2 host, the Debezium management UDFs are available in the `$HOME/asncdctools/src` directory. UDFs are available from the [Debezium examples repository](https://github.com/debezium/debezium-examples/tree/main/tutorial/debezium-db2-init/db2server).
+
+:::note
+If you have a docker instance of DB2, you can easily access as superuser with
+
+```shell
+docker exec -ti db2server bash -c "su - db2inst1"
+```
+
+We also suggest creating a DB2 sample with `db2sampl` command as it has been done [here](https://www.ibm.com/docs/en/db2/11.5?topic=linux-testing-your-db2-community-edition-docker-image-installation-systems#taskt_create_SAMPLEdb__steps__1) to actually create some files that could be needed for later steps.
+:::
+
+#### Database User and Permission Configuration
+
+Supposing you already have a DB2 instance running with the requirements explained above and superuser privileges, to put your tables into [Capture Mode](https://debezium.io/documentation/reference/2.1/connectors/db2.html#_putting_tables_into_capture_mode) you have to follow the following procedure:
+
+  1. Connect to the database `DB_NAME` you want to enable CDC on
+
+  ```shell
+  db2 connect to $DB_NAME
+  ```
+
+  2. Compile the Debezium management UDFs on the DB2 server host by using the `bldrtn` command provided with Db2:
+    
+    ```shell
+    cd $HOME/asncdctools/src
+    /opt/ibm/db2/V11.5/samples/c/bldrtn asncdc
+    ```
+
+  3. Start the database `DB_NAME` you want to monitor
+
+    ```shell
+    db2 start db $DB_NAME
+    ```
+
+  4. Make it possible for the JDBC driver to read the DB2 metadata catalog
+
+    ```shell
+    cd $HOME/sqllib/bnd
+    db2 bind db2schema.bnd blocking all grant public sqlerror continue
+    ```
+
+  5. Make a backup of `$DB_NAME`, because the ASN agent must have a starting point to read from (you can always set `BACK_UP_LOCATION=/dev/null`), then reconnect to it
```

Comment:
I would highlight that setting BACK_UP_LOCATION to /dev/null is advised only for development situations/purposes.

Reply:
Done